### ISSUE TITLE
Launching more than one instance of 'cliphist delete' will freeze bemenu

### ISSUE THREAD

**Issue by dogknowsnx** [sentriz/cliphist#24]
OS: Arch Linux
WM: wayfire, sway
Cliphist version: 0.3.0

How to reproduce:
Executing `cliphist list | bemenu  | cliphist delete` more than once/several times (via keybinding) will freeze `bemenu`.

Haven't tried piping `cliphist` into a different menu implementation...

Anyway, thanks for this little clipboard helper!

**Comment by sentriz** [comment]
howdy. is the situation improved with the latest commit?

**Comment by dogknowsnx** [comment]
No further freezes, so far...
EDIT: Will leave this one open and test with a growing database.
Of course feel free to close this if you cannot reproduce the issue (anymore)...

**Comment by sentriz** [comment]
cool thanks for trying it. i think the database size shouldn't change it that much so gonna close for now. feel free to reopen if needs be

**Comment by ThinkChaos** [comment]
Leaving a comment here because I don't think GH notifies when an issue gets mentioned in a PR. So here's a notification for you!

TL;DR: I have a proper fix, but can't build the latest commit so will need @sentriz to port the patch and release.
